media-libs/libsoundio/libsoundio-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit multilib cmake-utils
+
+DESCRIPTION="C99 library providing cross-platform audio input and output."
+HOMEPAGE="http://libsound.io/"
+SRC_URI="https://github.com/andrewrk/libsoundio/archive/${PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
I'd say it's better to use `http://libsound.io/release/libsoundio-1.0.0.tar.gz` here, because git(hub) tags are not as persistent as an upstream tarball. It has bitten us in the past, but that's a minor thing.